### DISCUSSION TITLE
Telescope: Filter by multiple tags

### DISCUSSION THREAD

**Discussion by ObaydaAlesawi** [laravel/framework#36372] [Q&A]
Is there any way to filter the telescope by multiple tags instead of one tag each time? 

![image](https://user-images.githubusercontent.com/32073999/108982686-1dfe1300-769f-11eb-909a-f14a2651935f.png)

**Comment by pgrimaud** [comment]
Unfortunately not (yet?*). Tags are not splitted and you can't combine them.

When you perform a search, I will trigger a SQL request like : 
```sql
select * from `telescope_entries` where `type` = ? and `uuid` in (select `entry_uuid` from `telescope_entries_tags` where `tag` = ?)
```

*You can open a PR and add this feature 🙌

**Reply by ObaydaAlesawi** [reply]
I can do the PR np, but will it be accepted as an Idea from @taylorotwell?

**Comment by mvehar-preskok** [comment]
Would this be implemented?

**Comment by AmirRezaM75** [comment]
@pgrimaud
Is it possible to use it with AND operator?
Right now, if we pass list of comma separated tags like (A,B)
It returns all entries which contains either A OR B

**Comment by cacosta-ui** [comment]
Sad, another feature discarded :(

**Comment by christhofer** [comment]
I have created the [pull request](https://github.com/laravel/telescope/pull/1726) to allow this
